docs/reference/data-frames/apis/put-transform.asciidoc

```diff
@@ -67,6 +67,15 @@ IMPORTANT:  You must use {kib} or this API to create a {dataframe-transform}.
 `source`::
   (object) Required. The source configuration, which consists of `index` and 
   optionally a `query`. See <<data-frame-transform-source>>.
+  
+`sync`::
+  (Optional, object) Defines the properties required to run continuously.
+  `field`:::
```

Comment:
there is another level between sync and field/delay, its "sync" -> "time" -> "field"/"delay" (see your example below)

sync defines continuous mode, time defines that the sync method is using a timestamp field in the data. In future we will have other ways to sync source and dest, e.g. for non timeseries data

Reply:
Thanks, I've drafted a description of that time property.